docs/source/references.bib

```diff
@@ -19,9 +19,20 @@ @article{mcinnes:18
   year={2018}
 }
 
+@misc{reuter:22,
+  author = {Reuter, Bernhard and Klein, Michal and Lange, Marius},
+  title = {pyGPCCA - python GPCCA: Generalized Perron Cluster Cluster Analysis package to coarse-grain
```

Comment:
@Marius1311, it seems that the capitalization is not correctly translated into in the final [reference](https://cellrank.readthedocs.io/en/latest/references.html#id20).